src/encoded/verifier.py

```diff
@@ -58,7 +58,7 @@ def verify_profile(item_type, indexer_testapp):
 @verifier
 def verify_schema(item_type_camel, registry):
     # test schema
```

Comment:
This was a call to a fixture but now calls a function that does what the fixture would have done.